src/viz/Range.js

```diff
@@ -0,0 +1,35 @@
+import * as THREE from 'three';
+
+import Core from '../core';
+import {DEFAULT_OPTIONS_POLYGON, MESSAGE_TYPE_RANGE} from '../utils/constants';
+import {DEFAULT_COLOR_ARROW} from '../utils/constants';
+import Group from '../primitives/Group';
+import Cone from '../primitives/Cone';
+
+class Range extends Core {
+  constructor(ros, topicName, options = DEFAULT_OPTIONS_POLYGON) {
+    super(ros, topicName, MESSAGE_TYPE_RANGE, options);
+    this.object = new Group();
+    this.updateOptions({
+      ...DEFAULT_OPTIONS_POLYGON,
+      ...options
+    });
+  }
+
+  updateOptions(options) {
+    super.updateOptions(options);
+    const { color, alpha } = this.options;
```

Comment:
what options exist in RViz?

Reply:
http://wiki.ros.org/rviz/DisplayTypes/Range

I guess the buffer length can be ignored for now